routes/orders.js

```diff
@@ -51,8 +55,8 @@ ordersRouter.patch(
   async (req, res, next) => {
```

Comment:
maybe url route needs to be: "/:orderId/:productId/quantity"...

Reply:
good point. however since an user can have only one active orderId, I am not sure about the /:orderId/:productId/quantity route yet. I'll keep in mind that and while we are working on front-end, I'll change the route to the /:orderId/:productId/quantity route if we need to do. Thanks :)